src/OpenTelemetry.Extensions/README.md

```diff
@@ -37,3 +37,33 @@ public static TracerProviderBuilder AddMyExporter(this TracerProviderBuilder bui
         .AddAutoFlushActivityProcessor(a => a.Parent == null && (a.Kind == ActivityKind.Server || a.Kind == ActivityKind.Consumer), 5000);
 }
 ```
+
+### BaggageSpanProcessor
+
+The BaggageSpanProcessor reads entries stored in Baggage from the parent context
+and adds the baggage keys and values to the activity as attributes on start.
+
+Add this span processor to a tracer provider.
+
+Warning!
+
+Do not put sensitive information in Baggage.
```

Comment:
mm..I actually think this warning belongs to the Baggage documentation, not this, right? This doc is merely copying values already in Baggage and putting it as Activity Tags.